### ISSUE TITLE
IMPORTANT: Please be sure to remove any private information before submitting.

### ISSUE THREAD

**Issue by juonjang** [microsoft/vscode-tools-for-ai#1631]
Does this occur consistently? 
Repro steps:

1.
2.

Action: azureAccount.onSessionsChanged
Error type: 123
Error Message: Unknown Error retrieving susbcriptions from Azure Account extension

Version: 0.16.0
OS: darwin
OS Release: 21.6.0
Product: Visual Studio Code
Product Version: 1.67.2
Language: en

<details>
<summary>Call Stack</summary>

```
_.<anonymous> extension.js:2:2029998
s extension.js:2:2026803
```

</details>

**Comment by jedcheng** [comment]
Hi there

This problem can be got around by signing in through the Azure plugin, instead of the Azure ML plugin.

Nonetheless, I got in touch with a Microsoft engineer about the exact same problem last month (on a Mac and signing in with Chrome).

After that, it seems that there is a recent update with VS code or Electron (I am not sure) that fixed this problem for signing in with Chrome.

We then tested it for Safari and Chrome to sign in. Using Safari doesn't help but using Chrome does work.
